### ISSUE TITLE
Any idea to support new GraalVM?

### ISSUE THREAD

**Issue by linux-china** [lucasallan/LuaTruffle#5]
GraalVM 1.0 is coming, and any idea to upgrade to last truffle api? 

```
 <groupId>com.oracle.truffle</groupId>
      <artifactId>truffle-api</artifactId>
      <version>1.0.0-rc2</version>
    </dependency>
```

**Comment by chrisseaton** [comment]
I think this project is pretty-much dead. It would be great to have a new effort to implement Lua on Truffle, and I would guess that because Lua is a relatively small language it should be fairly easy and hopefully competitive with LuaJIT.

**Comment by linux-china** [comment]
@chrisseaton  I agree your point.  For service mesh architecture, for example Envoy & Nginx mesh, these products support to write Lua code as rule or policy and push Lua code to agent for execution.  I just wonder if GraalVM has Lua Truffle support, and Java developers can reuse these Lua code for service mesh architecture with Java.     @chrisseaton  if possible could you guys do some modification and just make it work with new truffle-api?  you know it's open source, if some guys want  to use it in production and they can contribute their code.  Lua is still good as glue language for other languages. Thanks for your reply.
